AfterEffects/2018/index.d.ts

```diff
@@ -1,9 +1,10 @@
 /// <reference path="../../shared/global.d.ts" />
 /// <reference path="../../shared/ScriptUI.d.ts" />
 
-declare class Global {
-  app: Application
```

Comment:
why?

Reply:
1.  `app: Application` is already defined here in the global space: https://github.com/pravdomil/Types-for-Adobe/blob/master/shared/global.d.ts#L21
2. There isn't a way to reference this ^ Global class directly in AE, and so this isn't being used